scripts/deploy_env_scripts.py

```diff
@@ -0,0 +1,35 @@
+
+import json
+import os
+import shutil
+import sys
+
+"""
+    Creates env scripts to use PYTHONUSERSITE in our conda env
+    Following these instructions on how to set this up:
+    https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#macos-and-linux
+
+    Expected usage:
+    ```
+    conda activate openwpm
+    conda info --json | python deploy_env_scripts.py
+    conda activate openwpm
+    ```
+
+    This script aims to be plattform independent and idempotent
+"""
+
+CONDA_SCRIPT_DIR = "./etc/conda"
```

Comment:
I still have a path in my head of how we can support windows. I think it would be good if we didn't add things that inhibit that possibility.